config/jobs/kubernetes-sigs/cluster-api-provider-openstack/cluster-api-provider-openstack-periodics.yaml

```diff
@@ -96,3 +104,59 @@ periodics:
     testgrid-alert-email: sig-cluster-lifecycle-cluster-api-openstack-alerts@kubernetes.io
     testgrid-num-failures-to-alert: "2"
     testgrid-alert-stale-results-hours: "48"
+- name: periodic-cluster-api-provider-openstack-e2e-full-test-main
+  labels:
+    preset-service-account: "true"
+    preset-dind-enabled: "true"
+    preset-kind-volume-mounts: "true"
+  decorate: true
+  decoration_config:
+    timeout: 5h
+  interval: 12h
+  extra_refs:
+  - org: kubernetes-sigs
+    repo: image-builder
+    base_ref: master
+    path_alias: "sigs.k8s.io/image-builder"
+  spec:
+    containers:
+    - image: gcr.io/k8s-staging-test-infra/kubekins-e2e:v20230222-b5208facd4-master
+      env:
+      - name: "BOSKOS_HOST"
+        value: "boskos.test-pods.svc.cluster.local"
+      # The PR-Blocking tests are run in periodic-cluster-api-provider-openstack-e2e-test
+      # so we skip them here and run only the rest.
+      - name: E2E_GINKGO_SKIP
+        value: "\\[PR-Blocking\\]"
+      # These tests can use more than 2 machines per cluster and even 2 clusters
+      # per test for clusterctl upgrade tests, so we limit the parallel jobs
+      # to avoid capacity issues.
+      - name: E2E_GINKGO_PARALLEL
+        value: "1"
+      command:
+      - "runner.sh"
+      - "./scripts/ci-e2e.sh"
+      # we need privileged mode in order to do docker in docker
+      securityContext:
+        privileged: true
+      resources:
+        requests:
+          # these are both a bit below peak usage during build
+          # this is mostly for building kubernetes
+          memory: "9000Mi"
+          # during the tests more like 3-20m is used
+          cpu: 2000m
+  annotations:
+    testgrid-dashboards: sig-cluster-lifecycle-cluster-api-provider-openstack
+    testgrid-tab-name: periodic-e2e-full-test-main
+    testgrid-alert-email: sig-cluster-lifecycle-cluster-api-openstack-alerts@kubernetes.io
+    testgrid-num-failures-to-alert: "2"
+    testgrid-alert-stale-results-hours: "48"
+  reporter_config:
+    slack:
+      channel: "cluster-api-openstack"
+      job_states_to_report:
+      - failure
+      - aborted
```

Comment:
If aborted means 'we stopped it deliberately because somebody pushed an update the the PR' then lets not report that on Slack. Ideally one of these messages would be infrequent, surprising, and actionable.

Reply:
Periodics cannot be aborted by PRs I think. I would be worried to miss things like the job timing out and being aborted because of that (by prow). But I'm not sure about any of this :shrug: 
Do you think we should go with just `failure`?

Reply:
No, you're right. Lets leave it.